cli/src/utils/xlr/visitors/file.ts

```diff
@@ -0,0 +1,30 @@
+import type { Manifest } from '@player-tools/xlr';
+import path from 'path';
+import fs from 'fs';
+import type { VisitorProps } from './types';
+
+/** export all exported types in the file */
+export function fileVisitor(args: VisitorProps): Manifest | undefined {
+  const { sourceFile, converter, outputDirectory } = args;
+  const convertedTypes = converter.convertSourceFile(sourceFile);
+
+  if (convertedTypes.data.types.length === 0) {
+    return undefined;
+  }
+
+  const capabilities: Manifest = {
+    pluginName: 'Types',
```

Comment:
Why the hard-coded pluginName here?

Reply:
Whenever we export types in File mode we don't really have a good way of figuring out a plugin "name". Since the use case for this export is for core Player types we can set it statically.